src/sentry/lang/javascript/processor.py

```diff
@@ -520,6 +520,13 @@ def process_frame(self, processable_frame, processing_task):
         if not frame.get('abs_path') or not frame.get('lineno'):
             return
 
+        # can't fetch if this is internal node module as well
+        # therefore we only process user-land frames (starting with /)
+        # or those created by bundle/webpack internals
+        if self.data.get('platform') == 'node' and \
+            not frame.get('abs_path').startswith(('/', 'app:', 'webpack:')):
```

Comment:
Are we sure this is correct? There is no rule about which files might or might not appear in sourcemaps. For react-native i'm pretty sure we saw a bare 'index.bundle' as filename.

Reply:
I cannot give a definite answer, but I wasn't able to attain any other result than the one above and I tried a lot of variants.